packages/repco-webcomponent/README.md

```diff
@@ -0,0 +1,62 @@
+# repco-recommander-box
+
+A web component to show a list of recommended posts. This is still WIP and not ready for production right now.
+
+## Usage
+
+To use the `repco-recommander-box` component, add the following code to your HTML file:
+
+```html
+<script type="module" src="./dist/src/post-element.js"></script>
+<repco-recommander-box></repco-recommander-box>
+```
+
+Use the `post-element` tag in your HTML file. You can customize the display of the component by setting its attributes:
+
+- `count`: the number of posts to display (default is `10` also maximum right now)
+- `defaultThumbnail`: the default thumbnail image URL to display if a post does not have a thumbnail (default is ` `)
```

Comment:
this is `default-thumbnail` as HTML attribute (`defaultThumbnail`would be the DOM property then)